src/main/java/org/folio/service/invoice/PoLineInvoiceLineHolderBuilder.java

```diff
@@ -53,7 +55,7 @@ private CompletableFuture<List<InvoiceLine>> validateAndRetrievePaidInvoiceLines
       String poLineFundId = optionalFundDistribution.get().getFundId();
       return getCurrentFiscalYearInvoiceLines(poLineFundId, invoiceLines, requestContext)
         .thenApply(currentYearInvoiceLines -> {
-//          validateInvoiceLinesStatus(currentYearInvoiceLines);
+          validateInvoiceLinesStatus(currentYearInvoiceLines);
```

Comment:
Please leave this method commented. I will uncomment it in scope of second flow. Thanks

Reply:
Can we merge PR for integration tests and unblock work for UI if leave this validation commented?

Reply:
It's unreal from technically perspective

Reply:
okay, so you propose to merge this PR and related karate PR : https://github.com/folio-org/folio-integration-tests/pull/804  after Flow 2 PR will be merged

Reply:
okay so lets merge